deploy/charts/trust-manager/Chart.yaml

```diff
@@ -6,7 +6,7 @@ description: trust-manager is the easiest way to manage TLS trust bundles in Kub
 
 home: https://github.com/cert-manager/trust-manager
 
-kubeVersion: ">= 1.25.0"
+kubeVersion: ">= 1.25.0-0"
```

Comment:
Can you add a description that explains why we need this, eg. provide the "v1.19.13-eks-8df270" example.